test/fixtures.ts

```diff
@@ -448,4 +482,272 @@ export default {
+          },
+          {
+            path: {
+              ours: {
+                tuple: [`bip32:0'`],
+                string: `bip32:0'`,
+              },
+              theirs: `m/0'`,
+            },
+            privateKey:
+              '1559eb2bbec5790b0c65d8693e4d0875b1747f4970ae8b650486ed7470845635',
+            publicKey:
+              '0086fab68dcb57aa196c77c5f264f215a112c22a912c10d123b0d03c3c28ef1037',
+          },
+          {
+            path: {
+              ours: {
+                tuple: [`bip32:0'`, `bip32:2147483647'`],
+                string: `bip32:0'/bip32:2147483647'`,
+              },
+              theirs: `m/0'/2147483647'`,
+            },
+            privateKey:
+              'ea4f5bfe8694d8bb74b7b59404632fd5968b774ed545e810de9c32a4fb4192f4',
+            publicKey:
+              '005ba3b9ac6e90e83effcd25ac4e58a1365a9e35a3d3ae5eb07b9e4d90bcf7506d',
+          },
+          {
+            path: {
+              ours: {
+                tuple: [`bip32:0'`, `bip32:2147483647'`, `bip32:1'`],
+                string: `bip32:0'/bip32:2147483647'/bip32:1'`,
+              },
+              theirs: `m/0'/2147483647'/1'`,
+            },
+            privateKey:
+              '3757c7577170179c7868353ada796c839135b3d30554bbb74a4b1e4a5a58505c',
+            publicKey:
+              '002e66aa57069c86cc18249aecf5cb5a9cebbfd6fadeab056254763874a9352b45',
+          },
+          {
+            path: {
+              ours: {
+                tuple: [
+                  `bip32:0'`,
+                  `bip32:2147483647'`,
+                  `bip32:1'`,
+                  `bip32:2147483646'`,
+                ],
+                string: `bip32:0'/bip32:2147483647'/bip32:1'/bip32:2147483646'`,
+              },
+              theirs: `m/0'/2147483647'/1'/2147483646'`,
+            },
+            privateKey:
+              '5837736c89570de861ebc173b1086da4f505d4adb387c6a1b1342d5e4ac9ec72',
+            publicKey:
+              '00e33c0f7d81d843c572275f287498e8d408654fdf0d1e065b84e2e6f157aab09b',
+          },
+          {
+            path: {
+              ours: {
+                tuple: [
+                  `bip32:0'`,
+                  `bip32:2147483647'`,
+                  `bip32:1'`,
+                  `bip32:2147483646'`,
+                  `bip32:2'`,
+                ],
+                string: `bip32:0'/bip32:2147483647'/bip32:1'/bip32:2147483646'/bip32:2'`,
+              },
+              theirs: `m/0'/2147483647'/1'/2147483646'/2'`,
+            },
+            privateKey:
+              '551d333177df541ad876a60ea71f00447931c0a9da16f227c11ea080d7391b8d',
+            publicKey:
+              '0047150c75db263559a70d5778bf36abbab30fb061ad69f69ece61a72b0cfa4fc0',
+          },
+        ],
+      },
+    ],
+
+    // https://github.com/alepop/ed25519-hd-key/tree/d8c0491bc39e197c86816973e80faab54b9cbc26
+    // The state of the default branch as of 2021-10-19
+    'ed25519-hd-key': {
```

Comment:
Were these generated for testing using this tool? Because I can't find the fixtures referenced in the GitHub repo. If so, can we make that clearer?